perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -65,5 +62,8 @@ chmod +x perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 ./perfmetrics/scripts/presubmit/run_load_test_on_presubmit.sh
 sudo umount gcs
 
+# Executing integration tests
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/... -p 1 --integrationTest -v --testbucket=gcsfuse-integration-test
```

Comment:
how are GCSFuse logs useful here. I dont see we are storing them anywhere? can we clean the disk used by integration tests?

Reply:
If integration tests are failing, we can check the logs. Also, I checked size of the package. It was around 30 MB only. While building gcsfuse, I guess some hidden or other files are getting installed which is taking more space.

Reply:
where do we check the logs? if its on sponge2, then it has nothing to do with clearing the space consumed by package/integration tests.

Reply:
By doing ssh on kokoro VM. The VM is not getting deleted instantly. So one can check from there.

Reply:
thats the wrong way of checking logs. if someone doesn't keep track of execution, then there is no way to check the logs.